primitives/phragmen/src/lib.rs

```diff
@@ -141,12 +168,13 @@ pub fn elect<AccountId, FS>(
 	initial_candidates: Vec<AccountId>,
 	initial_voters: Vec<(AccountId, Vec<AccountId>)>,
 	power_of: FS,
-	total_power: Votes,
 ) -> Option<PhragmenResult<AccountId>>
 where
 	AccountId: Default + Ord + Member,
-	for<'r> FS: Fn(&'r AccountId) -> Votes,
+	for<'r> FS: Fn(&'r AccountId) -> Power,
```

Comment:
Is it good to replace Votes with Power?

Reply:
Yes, we pass `power_of` to this function.